lib/traject/macros/custom.rb

```diff
@@ -218,12 +218,19 @@ def hathi_to_hash(ht_format)
         CSV.read(hathi_overlap_csv)
            .group_by(&:shift)
            .each do |_oclc, ht_data|
-          ht_key = ht_format == 'mono' ? :ht_id : :ht_bib_key
-          ht_data.map! { |data| [ht_key, :access].zip(data).to_h }
+          ht_data.map! { |data| [ht_key(ht_format), :access].zip(data).to_h }
                  .uniq! { |data| data[:access] }
-          ht_data.reject! { |data| data[:access] == 'deny' } if ht_data.length > 1
+          ht_data.reject! { |data| data[:access] == ht_reject(ht_format) } if ht_data.length > 1
         end
       end
+
```

Comment:
```suggestion
      private

```

Since only other methods in this mixin send these messages I think they should be private.

Reply:
oh yes thanks! I thought about adding this then forget LOL!